src/city.py

```diff
@@ -49,11 +49,48 @@
 building_effects['Market'] = {'tax_score': 100, 'money_output': 1000, 'cost': 1500, 'size': 60}
 building_effects['Caravansary'] = {'population_capacity': 5, 'caravan_chance': 20, 'cost': 2000, 'size': 90}
 
+
+
+#Kenny - Additions
+building_effects['Small Hut'] = {'population_capacity': 1, 'tax_score': 1, 'cost': 1, 'size': 1}
+building_effects['Hut'] = {'population_capacity': 2, 'tax_score': 2, 'cost': 2, 'size': 2}
+building_effects['Long Hut'] = {'population_capacity': 4, 'tax_score': 5, 'cost': 5, 'size': 4}
+
+building_effects['Small House'] = {'population_capacity': 5, 'tax_score': 10, 'cost': 10, 'size': 5}
+building_effects['House'] = {'population_capacity': 10, 'tax_score': 20, 'cost': 50, 'size': 15}
+building_effects['Large House'] = {'population_capacity': 15, 'tax_score': 30, 'cost': 100, 'size': 25}
+
+building_effects['Small Villa'] = {'population_capacity': 20, 'tax_score': 50, 'cost': 200, 'size': 50}
+building_effects['Villa'] = {'population_capacity': 25, 'tax_score': 60, 'cost': 400, 'size': 60}
+building_effects['Large Villa'] = {'population_capacity': 40, 'tax_score': 70, 'cost': 800, 'size': 70}
+
+building_effects['Manor'] = {'population_capacity': 50, 'tax_score': 80, 'cost': 1600, 'size': 70}
+building_effects['Mansion'] = {'population_capacity': 60, 'tax_score': 90, 'cost': 3200, 'size': 80}
+building_effects['Castle'] = {'population_capacity': 100, 'weapons':10, 'train_bonus':10, 'tax_score': 100, 'cost': 6400, 'size': 100}
+
+building_effects['Bank'] = {'population_capacity': 8, 'money_output': 3000,  'cost': 1500, 'size': 20}
+
+building_effects['Guildhouse'] = {'population_capacity': 20, 'research_rate': 2, 'caravan_chance': 20, 'money_output': 500, 'cost': 4000, 'size': 50}
+building_effects['College'] = {'population_capacity': 20,  'research_rate': 3, 'cost': 3500, 'size': 50}
+building_effects['Inn'] = {'population_capacity': 50,  'tax_score': 5, 'money_output':500,'cost': 1000, 'size': 40}
+building_effects['Baracks'] = {'population_capacity': 15,  'morale_bonus': 25, 'train_bonus':5, 'weapons':2,'cost': 500, 'size': 30}
+building_effects['Blacksmith'] = {'population_capacity': 2,  'weapons': 10, 'cost': 350, 'size': 10}
+building_effects['Church'] = {'population_capacity': 5,  'tax_score': 5, 'cost': 200, 'size': 40}
+building_effects['Gun Smith'] = {'population_capacity': 5,  'gunpowder': 10, 'weapons':2, 'cost': 700, 'size': 50}
+building_effects['Sewer'] = {'population_capacity': 12,  'tax_score': 5, 'morale':25, 'cost': 1200, 'size': 35}
+building_effects['Graveyard'] = {'population_capacity': 2,  'morale': -5, 'gunpowder':5,'cost': 100, 'size': 30}
```

Comment:
Why does the graveyard produce gunpowder?

Reply:
I might have mistyped somethung